go/general_style_guide.md

```diff
@@ -127,7 +127,43 @@ You can run `nancy` locally by building or downloading the binary and running `g
 
 Please don't push secrets to GitHub unless they are encrypted.
 
-We recommend [configuring a client-side git hook](https://github.com/zricethezav/gitleaks/wiki/Scanning#uncommitted-changes-scan) so that your secrets are never pushed in the first place.
+We recommend [configuring a client-side git hook](https://github.com/zricethezav/gitleaks/wiki/Scanning#uncommitted-changes-scan) so that your secrets are never pushed in the first place. To do this:
+```
+# Create a directory to hold your global git hooks, if you don't have one
+$ mkdir /path/to/your/git-hooks
+
+# Change to the directory
+$ cd /path/to/your/git-hooks
+
+# Create a file named pre-commit
+$ touch pre-commit
+
+# Add the git hook content from the gitleaks wiki
+$ cat <<EOS >> pre-commit
+#!/bin/sh
+# Adds gitleaks as a pre-commit hook.
+
+gitleaksEnabled=\$(git config --bool hooks.gitleaks)
+cmd="gitleaks --verbose --redact --pretty"
+if [ "\$gitleaksEnabled" = "true" ]; then
```

Comment:
I don't think it will work if you don't run `git config hooks.gitleaks true` first

```suggestion
if [ "\$gitleaksEnabled" != "false" ]; then
```

Reply:
If the user hasn't set it then the value is just an empty string, not false